tests/stylus/StylintBearTest.py

```diff
@@ -0,0 +1,158 @@
+import os
+from queue import Queue
+
+from bears.stylus.StylintBear import StylintBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'stylint_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as f:
+        return f.readlines()
+
+
+@generate_skip_decorator(StylintBear)
+class StylintBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = StylintBear(Section('name'), Queue())
+
+    def test_bad_missing_colon(self):
+        filename = 'test_bad_missing_colon.styl'
+        file_contents = load_testfile(filename)
+        self.check_results(
+            self.uut,
+            file_contents,
+            [Result.from_values('StylintBear',
+                                message='missing colon between property '
+                                        'and value  colons',
```

Comment:
That was not so easy, but this works :)
```
(?P<line>\d+):?(?P<column>\d+)?\s+.*?(?P<severity>error|warning)\s+(?P<message>.+?)(  .*|\n|$)
```
Try to match a message until:
- Two spaces occur, and complete the match by consuming everything behind 2 spaces
- A new line occurs
- The end of string occurs (that might not terminate with \n, that's the reason for this special case)